web/client/plugins/GeoStory.jsx

```diff
@@ -53,7 +53,7 @@ const GeoStory = ({
 
     useEffect(() => {
         if (storyFonts.length > 0) {
-            const storyFontsSrc = storyFonts.filter(({ src }) => src && !document?.head.querySelector(`link[href='${src}']`));
+            const storyFontsSrc = storyFonts.filter(({ src }) => src && !(document && document.head).querySelector(`link[href='${src}']`));
```

Comment:
What is the scenario where document is undefined or null? And if it is, null.querySelector is going to crash anyway

Reply:
mmm, yes I was trying to replicate the exact thing, but it doesn't make to much sense.
```suggestion
            const storyFontsSrc = storyFonts.filter(({ src }) => src && !document.head.querySelector(`link[href='${src}']`));
```